ocs_ci/ocs/scale_lib.py

```diff
@@ -108,6 +110,8 @@ def create_multi_pvc_pod(
             io_runtime (seconds): Runtime in Seconds to continue IO
             pvc_size (int): Size of PVC to be created
             max_pvc_size (int): The max size of the pvc
+            remove_security_context_section (bool): If True, remove the security section from the Yaml file.
```

Comment:
let's keep it default as True, i.e. to remove security context section in yaml.

Reply:
Okay, np.